src/App.js

```diff
@@ -40,7 +40,7 @@ class App extends Component {
 
   componentDidCatch(error, info) {
     this.setState({hasError: true});
-    Raven.captureException(error, {extra: info}); //send error to raven client
+    //Raven.captureException(error, {extra: info}); //send error to raven client

```

Comment:
it's also commented out here? i'm guessing similar thing here with just comment by mistake?

Reply:
:man_facepalming: Thanks, @natibekele !